tests/test_openai_chat_director.py

```diff
@@ -509,3 +509,31 @@ async def test_get_correct_tools_with_chat_history(chat_client):
     # Assert
     tools = [tool.value for tool in tools]
     assert tools == ["online"]
+
+
+# ----------------------------------------------------------------------------------------------------
+@pytest.mark.anyio
+@pytest.mark.django_db(transaction=True)
+async def test_use_default_response_mode(chat_client):
+    # Arrange
+    user_query = "What's the latest in the Israel/Palestine conflict?"
+
+    # Act
+    mode = await aget_relevant_modes(user_query, {})
```

Comment:
These current tests should be moved into the `test_openai_chat_actor.py` tests

Chat director tests should test the response when calling the chat API. See `test_answer_using_file_filter` as example. We can check if the API response contains an image or text for the auto output tool selection tests you've added here.